templates/default/nginx.conf.erb

```diff
@@ -4,7 +4,7 @@ upstream disclosurebackend {
 
 server {
   listen 80;
-  server_name opencal.opendisclosure.io;
+  server_name opencal.opendisclosure.io disclosure.caciviclab.org;
```

Comment:
Yep, already covered. btw, we should be backing that up I guess. I'll add it to our backup issues.